Systemd: include network.target in After=

On a system where the `service` bridge is configured by NetworkManager albatross fails to restore VMs after a reboot due to the bridge not being up:

```
Jan 03 15:02:20 solsort albatrossd[1877]: albatrossd: [ERROR] failed to create [vm: stub-resolver]: interface service does not exist
```

Likely, adding `network.target` to the list in `After=...` in the systemd service files will fix this. I have added `network.target` on the machine and will confirm whether this fixed the issue (after I have done some upgrades).

--- a/README.md
+++ b/README.md
@@ -84,6 +84,7 @@ Init scripts for FreeBSD are provided in the `packaging/FreeBSD/rc.d`
 subdirectory, and a script to create a FreeBSD package
 `packaging/FreeBSD/create_package.sh`.
 For Linux, systemd service scripts are available in `packaging/Linux`.
+Consider removing `NetworkManager-wait-online.service` from the `Wants=` directive in albatross.service if you don't need it.
 
 It may help to read [the _outdated_ blog article](https://hannes.nqsb.io/Posts/VMM)
 for the motivation behind albatross and an overview of its functionality.

--- a/packaging/Linux/albatross_daemon.service
+++ b/packaging/Linux/albatross_daemon.service
@@ -1,7 +1,7 @@
 [Unit]
 Description=Albatross VMM daemon (albatrossd)
 Requires=albatross_console.socket albatross_daemon.socket
-After=syslog.target albatross_console.service
+After=syslog.target albatross_console.service network.target NetworkManager-wait-online.service
 
 [Service]
 Type=simple

